src/cli/dist.js

```diff
@@ -9,4 +9,5 @@
 require('../setup_node_env/dist');
 require('./apm')();
 require('../setup_node_env/root');
+require('../setup_node_env/mute_libraries');
```

Comment:
can't do it in `../setup_node_env/dist` because we don't want to import `moment` before APM, so I created a dedicated script/file for it. Please tell me if there was a better way.